src/data/item-desc.csv

```diff
@@ -1,71 +1,71 @@
+"中距離GⅠ・天皇賞（秋）を優勝した証。\n覚醒Lv強化で使用できる。","<size=22>G1 Tenno Sho (Autumn) victory proof. Used to raise Awakening Lv.</size>"
+"マイルGⅠ・マイルCSを優勝した証。\n覚醒Lv強化で使用できる。","<size=22>G1 Mile CS victory proof. Used to raise Awakening Lv.</size>"
+"中距離GⅠ・エリザベス女王杯を\n優勝した証。覚醒Lv強化で使用できる。","<size=22>G1 Queen Elizabeth II Cup victory proof. Used to raise Awakening Lv.</size>"
+"中距離GⅠ・ジャパンカップを優勝した証。\n覚醒Lv強化で使用できる。","<size=22>G1 Japan Cup victory proof. Used to raise Awakening Lv.</size>"
+"ダートGⅠ・チャンピオンズカップを\n優勝した証。覚醒Lv強化で使用できる。","<size=22>G1 Champions Cup victory proof. Used to raise Awakening Lv.</size>"
+"ジュニア級GⅠ・阪神JFを優勝した証。\n覚醒Lv強化で使用できる。","<size=22>G1 Hanshin JF victory proof. Used to raise Awakening Lv.</size>"
+"ジュニア級GⅠ・朝日杯FSを優勝した証。\n覚醒Lv強化で使用できる。","<size=22>G1 Asahi Hai FS victory proof. Used to raise Awakening Lv.</size>"
+"長距離GⅠ・有馬記念を優勝した証。\n覚醒Lv強化で使用できる。","<size=22>G1 Arima Kinen victory proof. Used to raise Awakening Lv.</size>"
+"ジュニア級GⅠ・ホープフルSを\n優勝した証。覚醒Lv強化で使用できる。","<size=22>G1 Hopeful Stakes victory proof. Used to raise Awakening Lv.</size>"
+"大事なレース日に遅刻は厳禁！育成の\n目標レースでコンティニューができる。","<size=22>No being late on the day of an important race! Lets you go back and continue from the last objective's race.</size>"
+"デイリーレースの挑戦に必要なチケット。","<size=22>A ticket necessary for participating in daily races.</size>"
+"レジェンドレースの挑戦に必要なチケット。","<size=22>A ticket necessary for participating in legend races.</size>"
+"トレーナー同士の友情の証。\nショップで様々なものと交換できる。","<size=22>Proof of the friendship between trainers. Can be exchanged at the shop for various items.</size>"
+"クラシック級GⅠ・ジャパンダートダービー\nを優勝した証。覚醒Lv強化で使用できる。","<size=22>G1 Japan Dirt Derby victory proof. Used to raise Awakening Lv.</size>"
+"ダートGⅠ・JBCスプリントを優勝した証。\n覚醒Lv強化で使用できる。","<size=22>G1 JBC Sprint victory proof. Used to raise Awakening Lv.</size>"
+"ダートGⅠ・JBCクラシックを優勝\nした証。覚醒Lv強化で使用できる。","<size=22>G1 JBC Classic victory proof. Used to raise Awakening Lv.</size>"
+"ダートGⅠ・JBCレディスクラシックを\n優勝した証。覚醒Lv強化で使用できる。","<size=22>G1 JBC Ladies Classics Winner victory proof's Garland. Used to raise Awakening Lv.</size>"
+"ダートGⅠ・帝王賞を優勝した証。\n覚醒Lv強化で使用できる。","<size=22>G1 Teio Sho victory proof. Used to raise Awakening Lv.</size>"
+"ダートGⅠ・東京大賞典を優勝した証。\n覚醒Lv強化で使用できる。","<size=22>G1 Tokyo Daishoten victory proof. Used to raise Awakening Lv.</size>"
+"能力を引き出す不思議な力。\nサポートカードのLv強化で使用できる。","<size=22>A mysterious power that draws out abilities. Used to increase the level of support cards.</size>"
+"サポートカードガチャが引けるチケット。\nガチャの一覧から使用できる。","<size=22>A support card gacha pull ticket. Usable at the Gacha menu.</size>"
+"★3確定メイクデビューガチャが引ける\nチケット。ガチャの一覧から使用できる。","<size=22>A ★3 guaranteed character gacha pull ticket. Usable at the gacha menu.</size>"
+"SSR確定メイクデビューガチャが引ける\nチケット。ガチャの一覧から使用できる。","<size=22>A SSR guaranteed support card gacha pull ticket. Usable at the gacha menu.</size>"
+"三女神様の祝福が得られるとされている。\nショップで様々なものと交換できる。","<size=22>It's said that the blessing of the three goddesses can be obtained from these. Can be exchanged for various items in the shop.</size>"
```

Comment:
This too

Reply:
The tickets should fit, as they're about as long as the JP text.
The goddess statue text definitely won't fit though, need somebody to test that.

Reply:
![5](https://user-images.githubusercontent.com/80930594/117278216-c512de00-ae8a-11eb-969c-9753e115ee61.png)
it doesnt fit